lib/featuresLoader.js

```diff
@@ -0,0 +1,90 @@
+const glob = require("glob");
+const path = require("path");
+const fs = require("fs");
+const { Parser } = require("gherkin");
+const log = require("debug")("cypress:cucumber");
+
+const { getStepDefinitionsPaths } = require("./getStepDefinitionsPaths");
+const { cucumberTemplate } = require("./cucumberTemplate");
+const { getCucumberJsonConfig } = require("./getCucumberJsonConfig");
+const {
+  isNonGlobalStepDefinitionsMode
+} = require("./isNonGlobalStepDefinitionsMode");
+
+const createCucumber = (
+  specs,
+  globalToRequire,
+  nonGlobalToRequire,
+  cucumberJson
+) =>
+  `
+    ${cucumberTemplate}
+    
+  window.cucumberJson = ${JSON.stringify(cucumberJson)};
+
+  ${globalToRequire.join("\n")}
+
+  ${specs
+    .map(
+      ({ spec, filePath, name }) => `
+        describe(\`${name}\`, function() {
+        ${nonGlobalToRequire &&
+          nonGlobalToRequire
+            .find(fileSteps => fileSteps[filePath])
+            [filePath].join("\n")}
+            
+        createTestsFromFeature('${filePath}', \`${spec}\`);
```

Comment:
think this would need to be 
`        createTestsFromFeature('${path.basename(filePath)}', \`${spec}\`);
`
to recreate the way cucumber.json files are currently generated - passing in the full path causes issues on windows boxes due to the c: part.

Reply:
hmmm ok! That leaves a possibility of feature names collisions (which I'd assumed was not a problem when every feature ran separately), but maybe that's not something to worry prematurely